docs/migrations/migrations.md

```diff
@@ -0,0 +1,232 @@
+: Plone security hotfixes will be made available for this series.
+  For more information, see the [security update policy](https://plone.org/security/update-policy).
+
+Currently, Plone 6.0.x is under maintenance support and Plone 5.2.x is under security support.
+That means new features are being developed for Plone 6.1.x, and bugfixes are being developed for Plone 6.0.x.
+
+Plone intends to provide stable releases (for example, 5.2.8 is a stable release of the 5.2.x series).
+Only bugfixes and changes that extend or improve an existing feature—as long as they don't break anything—make it into a bugfix release.
+
+
+### What's in a Plone version?
+
+In fact, it is even more complicated, because Plone consists of about 250 separate Python packages, all of which have their own versions and history.
+A Plone release such as Plone 5.2.6 (https://dist.plone.org/release/5.2.6/) pins all of these packages to a specific version, which altogether comprise a Plone version.
+
+Each Plone version has a detailed change log that lists all changes that made it into that release, as compared to the previous release.
+For an example see https://plone.org/download/releases/5.2.8.
+
+```{note}
+You can overrride the version of individual packages if a newer version has a feature or bugfix that you need.
+Then you may be using Plone version 5.2.6, but `plone.app.contenttype = 2.2.2` instead of `2.2.2` with its version pinned for Plone 5.2.6.
+You can do that, but at your own risk!
+Only the officially pinned version are tested with each other.
+```
+
+
+### Why do we need to upgrade?
+
+If you want to use a new feature or bugfix that your version does not yet have, then you need to upgrade.
+
+That could be as simple as changing an `extends` in your buildout.
+
+```ini
+extends = https://dist.plone.org/release/5.2.6/versions.cfg
+```
+
+Would become the following.
+
+```ini
+extends =  https://dist.plone.org/release/5.2.9/versions.cfg
+```
+
+But you not only need to change the version of Plone you use, but you also need to update the database.
+
+Since a Plone site stores content and settings in its database, these may need to be modified to reflect changes in the logic or configuration of Plone.
+This is done by upgrade steps, or code that runs and updates the database to work with the new version.
+
+
+## Update, Upgrade, Migration, or Relaunch?
+
+Changing from `5.2.6` to `5.2.9` is usually not a migration, but an update or upgrade, more specifically a bugfix update.
+
+Your could use the following:
+
+Hotfix-Upgrade
+: Adding a Hotfix to Plone without changing anything else.
+
+Update
+: Bugfix version changes, for example, `5.2.6` to `5.2.9` without additional changes.
+
+Upgrade
+: Bugfix or minor version change that gives you at least one new feature.
+  For example, `5.2.6` to `5.2.9`, while adding `collective.easyform`, or even `5.0.8` to `5.1.7` when you don't change anything else.
+
+Relaunch
+: The design, content structure, or major version changes.
```

Comment:
The design, graphical appearance/theme, used frontend libraries (like bootstrap) , content or content structure structure